entities/jackson/src/main/java/org/ccci/gto/globalreg/entity/jackson/Authentication.java

```diff
@@ -0,0 +1,233 @@
+
+package org.ccci.gto.globalreg.entity.jackson;
+
+import javax.annotation.Generated;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import org.apache.commons.lang.builder.EqualsBuilder;
+import org.apache.commons.lang.builder.HashCodeBuilder;
+import org.apache.commons.lang.builder.ToStringBuilder;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@Generated("org.jsonschema2pojo")
+@JsonPropertyOrder({
+    "id",
+    "key_guid",
+    "relay_guid",
+    "parent_id",
+    "created_by",
+    "updated_at",
+    "user_entered"
+})
+public class Authentication {
+
+    @JsonProperty("id")
+    private String id;
+    @JsonProperty("key_guid")
+    private KeyGuid keyGuid;
```

Comment:
the additional data it includes is used by SBR to figure out which record to return data from.

The model SBR produces and querying a single system produces should be identical, and that should be the model we support with this. Not the model for `filters[owned_by]=all`